.trivyignore

```diff
@@ -4,16 +4,12 @@
 CVE-2019-3826
 CVE-2022-27191
 
-#still present in mdsd telegraf
-CVE-2021-42836
-
-# ruby in /usr/lib
-CVE-2020-36327
-CVE-2021-43809
-CVE-2021-41816
-CVE-2021-41819
-CVE-2021-31799
-CVE-2021-28965
-
 #dpkg vulnerability in ubuntu
-CVE-2022-1304
\ No newline at end of file
+CVE-2022-1304
+
+# telegraf vulnerabilities
+CVE-2022-29190
```

Comment:
These Vulns will be fixed with other PR just merged, right? so we can remove them from ignore file.

Reply:
updated